### ISSUE TITLE
ASK Toolkit for VS Code - Only recently updated skills are appearing in Download list

### ISSUE THREAD

**Issue by talkingnews** [alexa/ask-toolkit-for-vscode#98]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Expected Behavior

"Download or edit skill" should provide a full list of all skills

## Current Behavior

Only appears to download skills edited/updated within last couple of months

## Steps to Reproduce (for bugs)

* Open ASK Toolkit for VSCode
* Go to "skills management > Download and Edit Skill"

It appears that only skills updated in developer console recently are listed.
I went to the developer console and simply did a build (nothing else) on all the skills. After that, the remaining skills appears in the list in vscode.

## Your Environment

* ASK SDK for Node.js used: 2.22.4
* Operating System and version: Ubuntu 20.04

## Node.js and NPM Info
* Node.js version used for development: 14.15.4
* NPM version used for development:  7.5.2

**Comment by nikhilym** [comment]
Hey @talkingnews , sorry for the inconvenience and thanks for reporting the issue. I just checked the `Download or edit skill` drop down and could confirm that all my `downloadable` skills are appearing in the drop down. By `downloadable`, I mean the skills are of `custom` type and are in `development`. 

We cache the skills locally to reduce the network calls. Did you try the `refresh` button on the right side of the drop down list, to explicitly refresh the skills list ? 

For example, here's my old skill from developer console

![Screen Shot 2021-02-08 at 9 42 05 AM](https://user-images.githubusercontent.com/25237515/107260065-9653ab80-69f2-11eb-88d0-ef0e537d10a4.png)

showing up in the toolkit drop down

![Screen Shot 2021-02-08 at 9 30 03 AM](https://user-images.githubusercontent.com/25237515/107260174-b6836a80-69f2-11eb-8010-6b7669d94a5a.jpeg)

---- 

- Can you try refreshing the skill list and let us know if that works for you? 
- Also, it will also help us debug the issue better if you can provide the toolkit version info. You can use the `Alexa: About Alexa Skills Toolkit` command from the command palette, to get the quick info we are looking for.

**Comment by talkingnews** [comment]
Hi @nikhilym . It looks like the Toolkit version has updated since my first post and is now

```
OS: Linux x64 5.4.0-1037-aws
Visual Studio Code Version:  1.53.0
Alexa Skills Toolkit Version: 2.3.0
Git Version: git version 2.25.1
```

Yes, I can confirm that I have refreshed the list. I think I have found the problem. It is only listing the 50 most recently updated skils. So if a developer has more than 50 skills, then they need to do a "build" if the skill they want to download is not appearing in the refreshed list.
